package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "napi-build-utils",
+  "version": "1.0.0",
+  "description": "A set of utilities to assist developers of tools building N-API native add-ons",
+  "main": "index.js",
+  "scripts": {
+    "doc": "./node_modules/.bin/jsdoc2md index.js >index.md",
```

Comment:
I believe that within the context of an npm `script` entry like this, the `./node_modules/.bin/` prefix can be omitted. That is, this command could look like:

`"doc": "jsdoc2md index.js > index.md"`

Reply:
Yes! I never knew that!